libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -770,6 +770,8 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "site_creation_category_viewed";
             case SITE_CREATION_THEME_VIEWED:
                 return "site_creation_theme_viewed";
+            case SITE_CREATION_DETAILS_VIEWED:
+                return "site_creation_details_viewed";
```

Comment:
Let's make sure this statistic is added to the tracking documentation.

Reply:
Done!